lib/states/az/mesa.R

```diff
@@ -8,39 +8,78 @@ source("common.R")
 # a little unclear as to which charges are specifically pedestrian vs.
 # vehicular, so our categorization here is weak; see outstanding TODO
 load_raw <- function(raw_data_dir, n_max) {
-  d <- load_single_file(
+  old_d <- load_single_file(
     raw_data_dir,
     "2014-03-17_citations_data_prr.csv",
     n_max
   )
-  bundle_raw(d$data, d$loading_problems)
+  
+  # adding source variable to use in 
+  # filtering out duplicates later
+  old_d$data <- old_d$data %>%
+    mutate(source = "old_data")
+  
+  updated_d <- load_single_file(
+    raw_data_dir, 
+    "2017_-_2019-09-23_Cites_PRR.csv", 
+    n_max
+  ) 
+  
+  new_d <- updated_d$data %>% 
+    mutate(
+      source = "new_data"
+    )
```

Comment:
Why create the `new_d` variable instead of just modifying `updated_d` the way you modified `old_d`?